xbmc/cores/VideoPlayer/VideoRenderers/OverlayRendererDX.cpp

```diff
@@ -229,30 +229,16 @@ COverlayImageDX::COverlayImageDX(CDVDOverlayImage* o)
 
   if(o->source_width && o->source_height)
   {
-    float center_x = (float)(0.5f * o->width  + o->x) / o->source_width;
-    float center_y = (float)(0.5f * o->height + o->y) / o->source_height;
+    float scareheight = (float)sourceHeight * o->source_width / sourceWidth;
```

Comment:
scareheight? scary name for this parameter.

Reply:
ah, i think about scaleheight here.

Reply:
may be change to name as scalevideoheight

Reply:
scareheight != scaleheight :)